cfg_mgmt/btp_application_certificate.py

```diff
@@ -0,0 +1,256 @@
+import copy
+import logging
+import typing
+import requests
+import tempfile
+import os
+from contextlib import contextmanager
+from dataclasses import dataclass
+from cryptography.hazmat.primitives import serialization
```

Comment:
`cryptography` package is referenced indirectly - please rather use `pycryptodome` (see requirements.txt)

```
pip show cryptography
Name: cryptography
Version: 36.0.2
Summary: cryptography is a package which provides cryptographic recipes and primitives to Python developers.
Home-page: https://github.com/pyca/cryptography
Author: The Python Cryptographic Authority and individual contributors
Author-email: cryptography-dev@python.org
License: BSD or Apache License, Version 2.0
Location: /usr/lib/python3.10/site-packages
Requires: cffi
Required-by: pdfminer.six, SecretStorage, aliyun-python-sdk-core-v3, paramiko, endesive, jwcrypto, homeassistant, pyOpenSSL
```

Reply:
There is no support for creating CSR in `pycryptodome`. I also don't find support for pkcs7.
Should I add `cryptography` as explicit requirement?

Reply:
yes, please :-)